lesson_07/dags/sql/transfer_from_dwh_bronze_to_dwh_silver_customers.sql

```diff
@@ -0,0 +1,30 @@
+TRUNCATE TABLE `{{ params.project_id }}.silver.customers`
+;
+
+INSERT `{{ params.project_id }}.silver.customers` (
+    first_name,
+    last_name,
+    email,
+    registration_date,
+    state,
+    client_id,
+
+    _logical_dt,
+    _job_start_dt
+)
+SELECT
+    FirstName                                    AS first_name,
+    LastName                                     AS last_name,
+    Email                                        AS email,
+    CAST(RegistrationDate AS DATE)               AS registration_date,
+    State                                        AS state,
+    MIN(Id)                                      AS client_id,
```

Comment:
The row with the same ID may be updated each day, so this trick doesn't work in our case.